### ISSUE TITLE
Swift 2.2 and enums defined between @interface and @end 

### ISSUE THREAD

**Issue by dmorrow** [ooyala/ios-sample-apps#80]
XCode 7.3 and Swift 2.2 don't like constants defined between @interface and @end.
https://forums.developer.apple.com/message/126002#126002
http://stackoverflow.com/questions/36148598/extern-nsstring-not-accessible-since-swift-2-2/36155592#36155592

`OOOoyalaPlayerControlType` is defined as an enum in `OOOoyalaPlayerViewController.h`.

``` objective-c
typedef NS_ENUM(NSInteger, OOOoyalaPlayerControlType) {
  /** an inline player, expandable to fullscreen */
  OOOoyalaPlayerControlTypeInline,
  /** a fullscreen player, not shrinkable to inline */
  OOOoyalaPlayerControlTypeFullScreen
};
```

Can you pull it out of the @interface block?

**Comment by lental** [comment]
sure thing! it'll be in the next release (4.13), coming out this month.

Thanks for the bug

**Comment by lental** [comment]
This is fixed in 4.13
